shards/shards.go

```diff
@@ -167,6 +214,8 @@ func NewDirectorySearcher(dir string) (zoekt.Streamer, error) {
 		return nil, err
 	}
 
+	go ss.watchPriorities(dir)
```

Comment:
you are not cleaning up after yourself here, it will just run forever. Please ensure you clean up nicely. Note that zoekt.Streamer has a Close function.